docs/First-Installation.md

```diff
@@ -58,24 +58,24 @@ The simplest way to get most of these components is through an all-in-one kit su
    * For macOS: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), [Apple Pi Baker](https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/), or the `dd` command
    * For Linux: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), or the `dd` command
     * MacOS/Linux users can optionally extract the .img image from the downloaded .img.gz by using `gunzip` (macOS users can also simply double-click it)
-4.  Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
-5.  Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
+4. Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
+5. Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
 
 ## Configure Controllers
 
-On first boot your filesystem will be expanded automatically, you will then be welcomed with the following screen- this menu will configure your controls for both EmulationStation and RetroArch Emulators:
+On first boot you will be welcomed with the screen below. This menu will configure your controls for both EmulationStation and all RetroArch emulators:
 
 ![welcomescreen](https://cloud.githubusercontent.com/assets/10035308/9140482/cf42f25c-3cee-11e5-8f91-c1fc1c57175c.png)
 
-Hold down any button on your keyboard or gamepad and the name will appear at the bottom and then open up into a configuration menu:
+Hold down any button on your controller or keyboard. While holding, its name will appear at the bottom of the window for a few seconds and then open up into a configuration menu:
 
-![welcomescreengamepadname](https://cloud.githubusercontent.com/assets/10035308/9140505/f5c19e38-3cee-11e5-965e-0e4e85ddaf56.png)
+![welcomescreengamepadconfigure](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png)
 
-Follow the onscreen instructions to configure your gamepad- if you run out of buttons just hold down a button to skip each unused button. **When you get to OK press the button you have configured as "A"**.
+Follow the onscreen instructions to configure your gamepad. If your controller doesn't have a button that you're being asked to define, just hold down any button to skip it.
```

Comment:
_define_ should be _configure_ for consistency.

Reply:
I figured [these images](https://cloud.githubusercontent.com/assets/10035308/9140518/0263b9c8-3cef-11e5-922f-42f790f3be91.png) made "define" more consistent because it's part of the UI ("Not defined")

Reply:
You're right, _define_ sounds closer to the on-screen text.